<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBAddDeviceDialogUi</class>
 <widget class="QWidget" name="OpenRGBAddDeviceDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add device</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SlidingStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="PageDeviceType">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;h2&gt;How is the device connected?&lt;/h2&gt;
Please make sure the device is plugged in before proceeding</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonUSB">
         <property name="text">
          <string>USB (or USB Dongle)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonMotherboard">
         <property name="text">
          <string>Motherboard (ARGB)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonGPU">
         <property name="text">
          <string>GPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRAM">
         <property name="text">
          <string>RAM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonLaptop">
         <property name="text">
          <string>Laptop (builtin keyboard)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonNetwork">
         <property name="text">
          <string>Network (WiFi)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonSerial">
         <property name="text">
          <string>Serial port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonDMX">
         <property name="text">
          <string>DMX</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageUSBType">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;h2&gt;What type of USB device is it?&lt;/h2&gt;
WARNING: This page should not be displayed. Detectors are NOT associated with device types and thus cannot be filtered by type.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="ButtonUSBTypeBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageScan">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="LabelScan">
         <property name="text">
          <string>&lt;h2&gt;Re-scanning devices...&lt;/h2&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="ButtonScanBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanRescan">
           <property name="text">
            <string>Scan again</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanNotFound">
           <property name="text">
            <string>Still not found</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanBeginning">
           <property name="text">
            <string>Add another device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonScanClose">
           <property name="text">
            <string>All good!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageSupportedList">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;h2&gt;Please check that your device is listed and enabled here&lt;/h2&gt;
Devices that are disabled, but are (possibly) connected are highlighted</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Filter:</string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Filter"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="TableSupported"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="ButtonSupportedBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonQMK">
           <property name="text">
            <string>Configure a QMK device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonSupportedNotFound">
           <property name="text">
            <string>Not listed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonSupportedRescan">
           <property name="text">
            <string>Rescan</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageNetworkType">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;h2&gt;Please select your device from the list below&lt;/h2&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonE131">
         <property name="text">
          <string>E1.31</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonKasaSmart">
         <property name="text">
          <string>Kasa Smart</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonLIFX">
         <property name="text">
          <string>LIFX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonPhilipsHue">
         <property name="text">
          <string>Philips Hue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonPhilipsWiz">
         <property name="text">
          <string>Philips Wiz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonYeelight">
         <property name="text">
          <string>Yeelight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonNanloeaf">
         <property name="text">
          <string>Nanoleaf</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonElgatoKeyLight">
         <property name="text">
          <string>Elgato KeyLight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonElgatoLightStrip">
         <property name="text">
          <string>Elgato LightStrip</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ButtonNetworkBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonNetworkNotFound">
           <property name="text">
            <string>Something else</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageConfigure">
      <layout class="QVBoxLayout" name="PageConfigureLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;h2&gt;Please configure your device here&lt;/h2&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="ConfigureEntry" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="ButtonConfigureBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonConfigureScan">
           <property name="text">
            <string>Save &amp;&amp; Scan</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageIDs">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;h2&gt;Let's look at hardware IDs&lt;/h2&gt;
Maybe you can spot your device in the list - it could help us identify the problem</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="ButtonIDsBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonIDsNotFound">
           <property name="text">
            <string>Not listed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Proceed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageHelp">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;h2&gt;Not the end of the world&lt;/h2&gt;
The device seems to be unsupported by OpenRGB - yet! With your help, we can fix that.
First, let's gather some information:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="helpIssueText">
         <property name="plainText">
          <string>Sample text for issue

```
Matching hardware IDs
```</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonClipboard">
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabelGitlab">
         <property name="text">
          <string>We would really appreciate if you could send us this info, by filing an issue on Gitlab. </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="ButtonHelpBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonHelpIssue">
           <property name="text">
            <string>File an issue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonHelpBeginning">
           <property name="text">
            <string>Try another device</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
